code/__DEFINES/is_helpers.dm

```diff
@@ -266,8 +266,7 @@ GLOBAL_LIST_INIT(glass_sheet_types, typecacheof(list(
 
 GLOBAL_LIST_INIT(book_types, typecacheof(list(
 	/obj/item/book,
-	/obj/item/spellbook,
-	/obj/item/storage/book)))
+	/obj/item/spellbook)))
```

Comment:
```suggestion
	/obj/item/spellbook,
)))
```

*mumble mumble* making the code better with every commit *mumble mumble* "you break it you fix it" *mumble grumble*